app.yaml

```diff
@@ -5,6 +5,12 @@ api_version: 1
 threadsafe: true
 
 handlers:
+- url: /wavedroplet.css
+  static_files: wavedroplet.css
+  upload: wavedroplet.css
+- url: /wavedroplet.js
+  static_files: wavedroplet.js
+  upload: wavedroplet.js
 - url: /d3viz.html(.*)
   static_files: d3viz.html
```

Comment:
not part of this patch but a side note: the .html extension is kind of ugly when it shows up in the URL.  We should actually take that out.  Also, there's no real reason to mention that it's a "d3" visualization in the URL.  Ideally URLs should refer to what they do, rather than implementation details.  So "view" or "visualize" would be a better name for this URL than d3viz.